redash/handlers/query_results.py

```diff
@@ -131,7 +132,7 @@ def post(self):
             'object_type': 'data_source',
             'query': query
```

Comment:
We should update the recorded event with the parameters if there are parameters.

Reply:
While we're at it, we should record the `query_id` as well.